test/MLOps.NET.Tests/Deployments/SchemaGeneratorTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.ML.Data;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MLOps.NET.Docker;
+using MLOps.NET.Docker.Interfaces;
+using MLOps.NET.Docker.Settings;
+using MLOps.NET.Entities.Impl;
+using MLOps.NET.Storage.Deployments;
+using Moq;
+using System.IO;
+using System.IO.Abstractions;
+using System.Threading.Tasks;
+
+namespace MLOps.NET.Tests.Deployments
+{
+    [TestClass]
+    public class SchemaGeneratorTests
+    {
+        private SchemaGenerator sut;
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            this.sut = new SchemaGenerator();
+        }
+
+        [TestMethod]
+        public void GenerateDefinition_ShouldGenerateModelOutput()
```

Comment:
Could we write a test here that checks that the outputed string matches **exactly** the class definition, including things like namespaces etc.? That would give us even more confidence that this will be doing what we need it to.